ingestion/src/metadata/orm_profiler/processor/orm_profiler.py

```diff
@@ -81,20 +82,18 @@ def __init__(
         self,
         config: ProfilerProcessorConfig,
         metadata_config: OpenMetadataConnection,
-        session: Session,
+        processor_interface: SQAProfilerInterface,
```

Comment:
This then could then be the generic ProfilerInterface I guess

Reply:
(I'll answer both comments) Yes, exactly. I am leaning more towards defining a protocol (vs an abc) and assigning that protocol as the type hint. I was thinking to wait to have the actual use case to implement but might be safer to have it right now.